backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/MoveOrCopyDiskCommand.java

```diff
@@ -173,7 +173,17 @@ && checkCanBeMoveInVm()
                 && checkIfNeedToBeOverride()
                 && setAndValidateDiskProfiles()
                 && setAndValidateQuota()
-                && validatePassDiscardSupportedForDestinationStorageDomain();
+                && validatePassDiscardSupportedForDestinationStorageDomain()
+                && isCopyWithLocalDc();
+    }
+
+    protected boolean isCopyWithLocalDc() {
+        if(getStoragePool().isLocal() && getStorageDomain().getStorageType() == StorageType.MANAGED_BLOCK_STORAGE
+                || getStorageDomain().getStorageType() == StorageType.MANAGED_BLOCK_STORAGE && getStoragePool().isLocal())  {
+            return failValidation(EngineMessage.ACTION_TYPE_FAILED_UNSUPPORTED_ACTION_FOR_MANAGED_BLOCK_STORAGE_TYPE);
```

Comment:
we need a better error to reflect this situation in which copy to/from MBS is blocked in local storage pool then

Reply:
ACK